### DISCUSSION TITLE
quick question: can you scroll up in the tulip REPL?

### DISCUSSION THREAD

**Discussion by diflouroethane** [shorepine/tulipcc#350] [General]
when typing a command (such as help(tulip) ) it puts output to the screen, but it passes too fast to see it all. is there a way to see the text that is not on the screen? or am I out of luck?

**Comment by bwhitman** [comment]
You cannot yet do this. We'd have to use more (precious) RAM to store a scrollback buffer. A workaround for now is to connect a computer and use the UART to show the output of the REPL.

**Comment by diflouroethane** [comment]
Ah. Ok. Thanks!
